Documentation/project-docs/eventpipe.md

```diff
@@ -0,0 +1,106 @@
+Using EventPipe
+===============
+
+# Capabilities #
+EventPipe provides the ability to collect traces using an in-process event logger instead of depending on a platform-level logger.  The following data can be collected:
+
+ - Sampled thread-time traces (Samples of every thread collected on a recurring basis at the specified (or default) rate)
+ - Runtime events (Events emitted by the native components of the runtime - e.g. GC, JIT, Interop, etc.)
+ - EventSource events
+ - Managed call stacks for all events.
+
+# Collection Examples #
+
+## Start/Stop Collection at Any Time with Default Collection Parameters ##
+
+1. Tell the runtime to listen for enable/disable commands:
+
+    > ```cmd
+    > export COMPlus_EnableEventPipe=4
+    > ```
+
+2. Specify the output file path:
+
+    > ```cmd
+    > export COMPlus_EventPipeOutputFile=/path/to/trace.netperf
+    > ```
```

Comment:
Note that there is a bit of an ugly problem if you end up running more than one .NET Process because this environment variable will be used by each process.    At best you lose the ability to collect data on the other processes (somebody wins).   

I would recommend changing things so that if the target file exists, you morph the name to <fileName>.PID.netperf (roughly what you do by default).   In the 'normal' case they get what they expected, but in the unusual case data is not lost.

Reply:
Fixed.  The control file name won't change - just the trace file itself (and only if the requested name already exists).